demo/demo.component.spec.ts

```diff
@@ -13,6 +12,7 @@ import {
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { By }           from '@angular/platform-browser';
 import { DebugElement } from '@angular/core';
+import { HttpClientModule } from '@angular/common/http';
```

Comment:
can you move this import on the place of the deleted HttpModule. On the top is the place for the angular dependencies, here are test dependencies. Can you fix this namespce placing for all components below.